app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,38 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = Food.where({ user_id: current_user.id })
+  end
+
+  def new
+    @food = Food.new
+  end
+
+  def create
+    name = food_params[:name]
+    measurement_unit = food_params[:measurement_unit]
+    quantity = food_params[:quantity]
+    price = food_params[:price]
+    food = Food.new(user_id: current_user.id, name:, measurement_unit:, price:, quantity:)
+
+    if food.save
+      redirect_to foods_path, notice: 'Food created successfully!'
+    else
+      render :new, notice: 'An error occured while creating a new food'
+    end
+  end
+
+  def destroy
+    @food = Food.find(params[:id])
+    if @food.destroy
+      redirect_to foods_path, notice: 'Food deleted successfully!'
+    else
+      redirect_to foods_path, alert: 'Unable to delete the food item.'
```

Comment:
Good implementation of create and delete actions in the food controller 👍, but by updating the routes.rb file by implementing the required changes below, I guess the path `foods_path` will change to something similar to `user_foods_path(@user.id)`.